examples/us-autocomplete-api/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/smartystreets/smartystreets-go-sdk/us-autocomplete-api"
+	"github.com/smartystreets/smartystreets-go-sdk/wireup"
+)
+
+func main() {
+	log.SetFlags(log.Ltime)
+
+	client := wireup.NewClientBuilder().
+		WithSecretKeyCredential(os.Getenv("SMARTY_AUTH_ID"), os.Getenv("SMARTY_AUTH_TOKEN")).
```

Comment:
Is it `WithSecretKeyCredential` rather than `WithSecretKeyCredentials` on purpose?